tests/features/java/java_s2i.feature

```diff
@@ -171,3 +171,7 @@ Feature: Openshift OpenJDK S2I tests
       Given s2i build https://github.com/jboss-container-images/openjdk from tests/OPENJDK-1549 with env using ubi9
        | variable           | value    |
        | MAVEN_ARGS         | validate |
+
+  Scenario: Ensure that run-env.sh placed in the JAVA_APP_DIR is sourced in the run script before launching java
+    Given s2i build https://github.com/jhuttana/openjdk-test-applications/tree/run_env_sh_test_coverage1 from quarkus-quickstarts/getting-started-3.0.1.Final-nos2i
```

Comment:
Same issue as https://github.com/jboss-container-images/openjdk/pull/405/files#r1370119574

Reply:
I think (this issue aside) the rest of this test is correct, but we may need to make more adjustments to the repository part. We need for run-env.sh, from the application source, to end up in $JAVA_APP_DIR (i.e., /deployments) after the S2I process has run. I'll have a think about how best to achieve this

Reply:
> Same issue as https://github.com/jboss-container-images/openjdk/pull/405/files#r1370119574

I have made the suggested change to URI usage in the "Given" section.

Reply:
> I think (this issue aside) the rest of this test is correct, but we may need to make more adjustments to the repository part. We need for run-env.sh, from the application source, to end up in $JAVA_APP_DIR (i.e., /deployments) after the S2I process has run. I'll have a think about how best to achieve this

Here : https://github.com/jhuttana/openjdk-test-applications/blob/pick_relevant_sources/getting-started/.s2i/environment the $JAVA_APP_DIR is set in environment file. Is it a good idea to use the same setup under https://github.com/jhuttana/openjdk-test-applications/tree/run_env_sh_test_coverage1/quarkus-quickstarts/getting-started-3.0.1.Final-nos2i  ?